aspnetcore/security/authentication/azure-ad-b2c.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Cloud authentication with Azure Active Directory B2C
+author: camsoper
+description: Shows how to set up Azure Active Directory B2C with ASP.NET Core
+keywords: ASP.NET Core,authentication,AAD,B2C,Azure Active Directory,Active Directory
+ms.author: casoper
+manager: wpickett
+ms.date: 01/12/2018
+ms.topic: article
+ms.technology: aspnet
+ms.prod: asp.net-core
+uid: security/authentication/azure-ad-b2c
+---
+# Cloud authentication with Azure Active Directory B2C
+
+By [Cam Soper](https://twitter.com/camsoper)
+
+[Azure Active Directory B2C](https://docs.microsoft.com/azure/active-directory-b2c/active-directory-b2c-overview) (AAD B2C) is a cloud identity management solution for your web and mobile applications.  The service provides global, scalable, cloud-based authentication to web applications and APIs using individual accounts, social network accounts, and federated enterprise accounts for applications hosted in the cloud and on-premises.  Additionally, AAD B2C can provide multifactor authentication with very little configuration.
+
+This page demonstrates using Azure Active Directory B2C with ASP.NET Core.
+
+## Prerequisites
+
+The following are required for this walkthrough:
+
+* A [Microsoft Azure subscription](https://azure.microsoft.com/free/?ref=microsoft.com&utm_source=microsoft.com&utm_medium=docs&utm_campaign=visualstudio).  
+* [Visual Studio 2017](https://www.visualstudio.com/vs/) (any edition).
+
+## Create the Azure Active Directory B2C tenant
+
+Create an Azure Active Directory B2C tenant [as described in the documentation](https://docs.microsoft.com/azure/active-directory-b2c/active-directory-b2c-get-started).  When prompted, associating the tenant with an Azure subscription is optional for this tutorial.
+
+## Register the application in AAD B2C
+
+In the newly created AAD B2C tenant, register your application using [the steps in the documentation under the *Register a web app* heading](https://docs.microsoft.com/azure/active-directory-b2c/active-directory-b2c-app-registration#register-a-web-app). Stop at the *Create a web app client secret* heading. A client secret isn't required for this tutorial. 
```

Comment:
I don't know what our actual guidelines are about referring to the word "heading" when referencing a heading, but I leave the word "heading" out and say something like "See XYZABCWhateves in XYZABC article"
Doesn't bother me though if this stays or goes in the end.

Reply:
I think we usually refer to is as section.